src/streaming/utils/TTMLParser.js

```diff
@@ -858,19 +872,23 @@ function TTMLParser() {
         if ('background-color' in cueStyle) {
             if (cueStyle['background-color'].indexOf('#') > -1 && (cueStyle['background-color'].length - 1) === 8) {
                 rgbaValue = convertHexToRGBA(cueStyle['background-color']);
-                properties.push('background-color: ' + rgbaValue);
-            } else {
-                properties.push('background-color:' + cueStyle['background-color'] + ';');
+            } else if (cueStyle['background-color'].indexOf('rgba') > -1) {
+                rgbaValue = convertAlphaValue(cueStyle['background-color']);
+            }  else {
+                rgbaValue = cueStyle['background-color'] + ';';
```

Comment:
for the rgb use case, we don't have to convert data. I have update this PR in order to fix test for alpha value greater than 1.